helix-view/src/clipboard.rs

```diff
@@ -138,6 +169,9 @@ pub fn get_clipboard_provider() -> Box<dyn ClipboardProvider> {
             paste => "termux-clipboard-get";
             copy => "termux-clipboard-set";
         }
+    } else if tmux_allow_passthrough() {
+        // If we are in tmux and it allows passthrough, lets use OSC52 and escape tmux.
+        Box::new(provider::FallbackProvider::new())
     } else if env_var_is_set("TMUX") && binary_exists("tmux") {
```

Comment:
Why is this a separate branch rather than nesting under the next one that's already checking for tmux?

Reply:
I don't know why I did that. I combined the the two branches.